tasks/move-granules/index.js

```diff
@@ -293,6 +293,13 @@ async function moveGranules(event) {
   // first we check if there is an output file
   const config = event.config;
   const bucketsConfig = new BucketsConfig(config.buckets);
+
+  const bucketTypes = Object.values(bucketsConfig.buckets)
+    .reduce(
```

Comment:
Nit:  It might be nice to DRY this logic a bit, as we're using it repeatedly for a similar purpose to pull out a name->type relationship.  That said, it's a one line loop and I can see pulling in common all over for that has other downsides.     Just something to consider, not going to hold up the PR on it.

Reply:
Yeah, I thought about that. It's used in `api` and `cmrjs`, which meant putting it somewhere they could both reach. I decided that, in this case, it wasn't worth the overhead.